CentralizedConsentAPI/README.md

```diff
@@ -0,0 +1,154 @@
+# Centralized Consent API
+
+## Authors:
+
+- [Kate Cheney](https://github.com/kcheney1)
+
+## Table of Contents
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+
+- [Definitions](#definitions)
+- [Introduction](#introduction)
+  - [Goals](#goals)
+- [JavaScript API](#javascript-api)
+  - [Requirements](#requirements)
+- [Scenario](#scenario)
+- [Detailed design discussion](#detailed-design-discussion)
+  - [Offering an opt-in preference](#offering-an-opt-in-preference)
+  - [Default value](#default-value)
+  - [Make per-site and global preferences indistinguishable](#make-per-site-and-global-preferences-indistinguishable)
+  - [Limiting consent information to two bits](#limiting-consent-information-to-two-bits)
+  - [Legal effects](#legal-effects)
+- [Stakeholder Signals](#stakeholder-signals)
+- [Acknowledgements](#acknowledgements)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Definitions
+
+**Consent banner:** Prompt or banner on a site informing a user what data is
+being collected on them and how it is being used, and providing options to
+manage data selling/sharing preferences.
+
+## Introduction
+
+The Centralized Consent API aims to reduce the need for a website to display
+consent banners while still complying with consumer privacy legislation.
+The browser will store the user's global data selling/sharing preference, and
+optionally per-site preferences. The browser will offer 2 bits of per-site storage
+if no preference is set. With the Centralized Consent API, a site only needs to
+display a consent banner if the user preference has not been set, resulting in
+fewer banners. Additionally, the per-site storage will be more persistent than
+cookies, decreasing the frequency of new banners after website data is cleared.
+
+### Goals
+
+* Lessen the frequency of consent banners on sites to improve user experience.
+  Frequent banners obstruct web content and disrupt user flow.
+* Provide users more practical control over the use of their personal data.
+  Requiring a user to opt-in/out of personal data selling/sharing on a per-site
+  basis, often via a multi-step process, does not always provide the user with
+  practical autonomy over their data.
+
+## JavaScript API
+
+* `document.dataConsentPreference():` Returns a promise with the user's
+  personal data selling/sharing preference. Rejects the promise if the user's
+  preference is unknown.
+* `document.setDataConsentInformation():` Sets up to two bits of consent
+  information if a user has not set a preference. This will be cleared on a
+  rolling 90 day basis with user interaction.
```

Comment:
Without user interaction? (If a user does not interact with the site for 99 days, the consent information will be cleared?)

Reply:
Good catch, this could probably even be worded more clearly. I'll change it.